ddtrace/contrib/requests/connection.py

```diff
@@ -58,7 +58,10 @@ def _wrap_request(func, instance, args, kwargs):
 
     with tracer.trace("requests.request", span_type=http.TYPE) as span:
         # update the span service name before doing any action
-        span.service = _extract_service_name(instance, span, netloc=parsed_uri.netloc)
+        hostname = parsed_uri.hostname
+        if parsed_uri.port:
+            hostname += ":{}".format(parsed_uri.port)
```

Comment:
Sorry I didn't catch this before, but this is the last little change then I'll merge!

```suggestion
            hostname += ':{}'.format(parsed_uri.port)
```